packages/verification-app/.env

```diff
@@ -7,4 +7,4 @@ VUE_APP_CLAIM_CONTRACT_ADDRESS=
 VUE_APP_ACCEPTED_ISSUER_KEY=
 VUE_APP_CERTIFACTION_API_URL=
 VUE_APP_CERTIFACTION_CDN_BASE_URL=
-VUE_APP_PDF_WASM_VERSION=1.6.17
+VUE_APP_PDF_WASM_VERSION=1.6.22
```

Comment:
HIN multisign changes are behind v2.0.0+. So it would probably make sense to update it to the newest version but maybe there's a reason to not do it now. @bauer00 has more insight and I would wait also for his feedback

Reply:
Yes, would make sense, but as it probably also needs changes in the wasm wrapper, worker and service, a separate branch would be better for that.

Reply:
Alessandro updated the verification package to the latest version, so please rebase or merge the latest changes into this branch.